.github/workflows/update-gradle-wrapper.yml

```diff
@@ -0,0 +1,25 @@
+name: Update Gradle Wrapper
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
```

Comment:
This should be increased to once a month, otherwise it'll run too often for no reason.

Reply:
> This should be increased to once a month, otherwise it'll run too often for no reason.

I think it's good to check every day to use the latest version as early as possible but I changed it as you requested. Anyway, i added `workflow_dispatch` so you can manually run them if there is a gradle wrapper update.